zerver/lib/event_queue.py

```diff
@@ -463,9 +463,9 @@ def load_event_queues():
 
 def send_restart_events(immediate=False):
     # type: (Optional[bool]) -> None
-    event = dict(type='restart', server_generation=settings.SERVER_GENERATION)
+    event = dict(type='restart', server_generation=settings.SERVER_GENERATION)  # type: Dict[text_type, Any]
```

Comment:
I think this should be `Dict[str, Any]`, since keyword argument names are `str`s.